core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -35,6 +35,8 @@
   </div>
 </div>
 
+<on-screen-keyboard customizable="false" custom-letters="''"></on-screen-keyboard>
```

Comment:
So this is how it's supposed to function:
The math interactions will all have a customization arg for specifying what letters should be displayed to the learner. This data will be passed into the component during initialization. But adding this customization arg requires a state migration which is why I've decided to do it separately. So once the cust arg is implemented and added to the interactions, the `customizable` field will be removed and only the `custom-letters` field will be present so that we can pass in which the letters to be displayed. Currently, it's just for checking if the UI works as expected. Please let me know if you'd like me to explain something in more detail :)